packages/cosmic-swingset/src/chain-main.js

```diff
@@ -553,11 +553,11 @@ export default async function main(progname, args, { env, homedir, agcc }) {
           },
         );
 
-        exportData.exporter.onDone().catch(() => {
+        exportData.exporter.onDone().catch(async () => {
           if (exportData === stateSyncExport) {
             stateSyncExport = undefined;
           }
-          exportData.cleanup();
+          await exportData.cleanup();
```

Comment:
cleaner to just return the promise:
```suggestion
          return exportData.cleanup();
```
Note that if the cleanup fails it will be an unhandled promise rejection. Seems okay tho

Reply:
Done